app/components/channel_link/channel_link.test.js

```diff
@@ -58,18 +65,46 @@ describe('ChannelLink', () => {
     });
 
     test('should call props.actions and onChannelLinkPress on handlePress', () => {
+        const actions = {...baseProps.actions, joinChannel: jest.fn()};
```

Comment:
This actually isn't needed since functions mocked using `jest.fn()` have their internal state reset between tests